tools/redirect_cc/redirect_cc.cc

```diff
@@ -149,26 +132,31 @@ class RedirectCC {
             // Remove android_clang_arm64/gen/ prefix.
             path_cc.remove_prefix(path_cc_parts[0].size() + 1);
             path_cc.remove_prefix(path_cc_parts[1].size() + 1);
+          } else {
+            LOG(WARNING) << "unhandled path: " << path_cc;
+            path_cc = base::FilePath::StringPieceType();
           }
         }
 
-        base::FilePath::StringType brave_path = base::StrCat(
-            {brave_chromium_src_dir, FILE_PATH_LITERAL("/"), path_cc});
-        if (base::PathExists(base::FilePath(brave_path))) {
-          launch_argv.push_back(base::FilePath::StringType(arg_piece));
-          launch_argv.push_back(std::move(brave_path));
-          ++arg_idx;
-          continue;
+        if (!path_cc.empty()) {
+          base::FilePath::StringType brave_path = base::StrCat(
```

Comment:
Better to use FilePath::Append, isn't it ?

Reply:
here we need to force forward slash even on windows, that's why direct strcat. goma cross-compiles on linux and expects `/c` to be always forward-slashed.